resolvers/node/resolution.rs

```diff
@@ -63,6 +64,35 @@ fn conditions_from_resolution_mode(
   }
 }
 
+pub struct ConditionsFromResolutionMode {
+  pub func: Box<
```

Comment:
I don't think we need to expose this property?

Reply:
Done